src/Relay/Relay.Test/ScenarioTests/HybridConnectionsTests.ps1

```diff
@@ -71,7 +71,7 @@ function HybridConnectionsTests
 	
 		# Update the Created HybridConnections
 		Write-Debug " Update HybridConnections "
-		$createdHybridConnections.UserMetadata = "usermetadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."	   
+		$createdHybridConnections.UserMetadata = "user metadata is a placeholder to store user-defined string data for the HybridConnection endpoint.e.g. it can be used to store  descriptive data, such as list of teams and their contact information also user-defined configuration settings can be stored."	   
```

Comment:
@nschonni I don't think this change should be made, but we should double-check

(this comment applies to all instances of this change in this PR)

Reply:
OK, I was going by the camel casing userMetadata that I came across, but i'll drop the commit for now